src/platform/packages/shared/kbn-esql-ast/src/types.ts

```diff
@@ -241,6 +242,7 @@ export type BinaryExpressionRegexOperator = 'like' | 'not_like' | 'rlike' | 'not
 export type BinaryExpressionRenameOperator = 'as';
 export type BinaryExpressionWhereOperator = 'where';
 export type BinaryExpressionMatchOperator = ':';
+export type BinaryExpressionIn = 'in' | 'not_in';
```

Comment:
Should be:

```suggestion
export type BinaryExpressionIn = 'in' | 'not in';
```

in line with latest changes

Reply:
@vadimkibana I think we should do it, no?